src/oas.md

```diff
@@ -1183,11 +1273,14 @@ content:
     schema:
       type: string
     example: $.a.b[1:1]
+examples:
```

Comment:
`examples` and `example` can currently only be used with `schema`, not with `content`, see https://github.com/OAI/OpenAPI-Specification/blob/v3.2-dev/src/oas.md#fixed-fields-for-use-with-content.

This affects multiple examples :sad: